UsbAccess/UsbAccess.cpp

```diff
@@ -769,5 +1064,71 @@ namespace Plugin {
 
         return result;
     }
+
+    uint32_t UsbAccess::getUSBSizeWrapper(const JsonObject &parameters, JsonObject &response)
+    {
+        LOGINFOMETHOD();
+
+        bool result = false;
+        char cmd[128] = {'\0'};
+        char totalsize[20] = {'\0'};
+        char usedsize[20] = {'\0'};
+        char freesize[20] = {'\0'};
+
+        string pathParam;
+        if (parameters.HasLabel("path"))
+        {
+            pathParam = parameters["path"].String();
+        }
+        else
+        {
+            response["error"] = "don't specify the mount path";
+            returnResponse(result);
+        }
+        snprintf(cmd, 127, "df -h | grep %s$ | awk \'{print $2 \" \" $3 \" \" $4}\'", pathParam.c_str());
+        FILE* fp = popen(cmd, "r");
+        if(fp)
+        {
+            if(fscanf(fp,"%s %s %s", totalsize, usedsize, freesize) == EOF)
+            {
+                returnResponse(result);
+            }
+            response["totalsize"] = string(totalsize);
+            response["usedsize"] = string(usedsize);
+            response["freesize"] = string(freesize);
+            pclose(fp);
+            result = true;
+        }
+        returnResponse(result);
+    }
+
+    uint32_t UsbAccess::unmountUSBWrapper(const JsonObject &parameters, JsonObject &response)
+    {
+        LOGINFOMETHOD();
+
+        bool result = false;
+        char cmd[128] = {'\0'};
+
+        string pathParam;
+        if (parameters.HasLabel("path"))
+        {
+            pathParam = parameters["path"].String();
+        }
+        else
+        {
+            response["error"] = "don't specify the mount path";
+            returnResponse(result);
+        }
+        snprintf(cmd, 127, "systemd-umount -u %s", pathParam.c_str());
+        FILE* fp = popen(cmd, "r");
```

Comment:
Since the popen is used on the data coming from outside, v_secure_popen() should be used for handling more secure way.

Reply:
Done